packages/shared/components/molecules/activity-info/SmartContractActivityInformation.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="typescript">
+    import { KeyValueBox } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { NftActivity } from '@core/wallet'
+    import { parseLayer2Metadata } from '@core/layer-2'
+
+    export let activity: NftActivity
```

Comment:
i suppose this should be either `Activity` if all are allowed or `BasicActivity` if only normal transactions are allowed?

Reply:
For now only `BasicActivity` is allowed. In the future we will allow more actvities.